docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,100 @@
+# Getting Started
+
+## What you'll do
+
+- Deploy the [jenkins-agent charm](https://charmhub.io/jenkins-agent)
+- Deploy the [jenkins-k8s charm](https://charmhub.io/jenkins-k8s) and integrate with it via a cross-model integration
+
+The `jenkins-agent` charm helps deploy a Jenkins agent with ease and also helps operate the charm. This
+tutorial will walk you through each step of deployment to get a basic Jenkins agent deployment and integrate it with Jenkins.
+
+### Prerequisites
+
+To deploy the `jenkins-agent` charm, you'll need to have a bootstrapped machine model. Learn about
+bootstrapping different clouds [here](https://juju.is/docs/olm/get-started-with-juju#heading--prepare-your-cloud).
+
+Use `juju bootstrap localhost localhost` to bootstrap a `lxd` machine controller with the name
+`localhost` for tutorial purposes.
+
+### Setting up the tutorial model
+
+To easily clean up the resources and to separate your workload from the contents of this tutorial,
+it is recommended to set up a new model with the following command.
+
+```
+juju add-model tutorial
+```
+
+### Deploy the jenkins-agent charm
+
+Start off by deploying the jenkins-agent charm. By default it will deploy the latest stable release
+of the jenkins-agent charm.
+
+```
+# Deploy an edge version of the charm until stable version is released.
```

Comment:
Is the below command intended to deploy the edge version? It seems to be using the stable version?

Reply:
I updated the command with `--channel=latest/edge`